src/app/lib/config/toggles/liveConfig.js

```diff
@@ -26,11 +26,7 @@ export default {
   include: {
     enabled: true,
   },
-  liveRadioSchedule: {
-    enabled: true,
-    value:
-      '(afaanoromoo|afrique|amharic|arabic|bengali|burmese|hausa|indonesia|korean|nepali|pashto|persian|somali|swahili|tigrinya|urdu)',
-  },
+
```

Comment:
We'll need to add the liveRadioSchedule toggle to the live environment before we can push this PR - I have access

Reply:
Done! 🚀